src/Raven.Client/ServerWide/Tcp/TcpConnectionHeader.cs

```diff
@@ -28,7 +28,7 @@ public enum OperationTypes
         public static readonly int NumberOfRetriesForSendingTcpHeader = 2;
         public static readonly int ClusterTcpVersion = 10;
         public static readonly int HeartbeatsTcpVersion = 20;
-        public static readonly int ReplicationTcpVersion = 41;
+        public static readonly int ReplicationTcpVersion = 31;
```

Comment:
I don't understand, how can you negotiate with the other side if you keep the same version?